File: app/src/androidTest/java/net/tlalka/android/fiszki/view/pages/TestListPage.java
package net.tlalka.android.fiszki.view.pages;

import net.tlalka.android.fiszki.R;

public class TestListPage extends AbstractPage {

    @Override
    public void valid() {
        isVisible(R.id.test_icon);
        isVisible(R.id.test_topic);
        isVisible(R.id.test_progress);
        isVisible(R.id.test_list_view);
    }

    public void openItem(int index) {
        clickListItem(R.id.test_list_view, index);
    }
}

File: app/src/androidTest/kotlin/net/tlalka/fiszki/view/pages/TestListPage.kt
package net.tlalka.fiszki.view.pages

import net.tlalka.fiszki.R

class TestListPage : AbstractPage() {

    override fun valid() {
        isVisible(R.id.test_icon)
        isVisible(R.id.test_topic)
        isVisible(R.id.test_progress)
        isVisible(R.id.test_list_view)
    }

    fun openItem(index: Int) {
        clickListItem(R.id.test_list_view, index)
    }
}
